src/main/java/com/qunite/api/web/mapper/QueueMapper.java

```diff
@@ -12,13 +14,12 @@
 @Mapper(unmappedTargetPolicy = ReportingPolicy.IGNORE)
 public interface QueueMapper {
   @Mapping(source = "creatorId", target = "creator.id")
-  Queue toEntity(QueueDto queueDto);
+  Queue toEntity(QueueCreationDto queueDto);
 
   @Mapping(source = "creator.id", target = "creatorId")
   @Mapping(target = "createdAt", dateFormat = "dd-MM-yyyy HH:mm:ss")
   QueueDto toDto(Queue queue);
 
   @BeanMapping(nullValuePropertyMappingStrategy = NullValuePropertyMappingStrategy.IGNORE)
-  @Mapping(source = "creatorId", target = "creator.id")
```

Comment:
Why did you remove that line of code?

Reply:
Field "creatorId" is not present in QueueUpdateDto. In my opinion, it is wrong to have opportunity to change creator of the queue.